.github/workflows/jib-gradle-prepare-release.yml

```diff
@@ -41,24 +64,38 @@ jobs:
         with:
           # Provided by Actions. No need to create a token.
           github_token: ${{ secrets.GITHUB_TOKEN }}
-          source_branch: gradle_release_v${{ github.event.inputs.release_version }}
-          pr_title: "Gradle release v${{ github.event.inputs.release_version }}"
+          source_branch: ${{ github.event.inputs.project }}-release-v${{ github.event.inputs.release_version }}
+          pr_title: "${{ github.event.inputs.project }} release v${{ github.event.inputs.release_version }}"
           pr_body: "To be merged after the release is complete."
 
-      - name: Draft GitHub release
+      - name: Draft Maven/Gradle GitHub release
```

Comment:
Just to clarify, we don't yet draft releases for the extension projects?

Reply:
Yeah, we never created GitHub releases for these. (We could have.)